ravindu14/SliitSpmProject
# [task-1017] Supervisor's interface for form 1 is done

How to Test
1. switch to branch feature/task-1017-create-interface-for-form1
2. go to supervisor form tab
3. Then the interface to download and upload the form is displayed.
